dir/path.go

```diff
@@ -69,27 +89,38 @@ func (p *PathManager) Localkey(name string) (keyPath, certPath string) {
 	return keyPath, certPath
 }
 
-// SigningKeyConfig return the path of signingkeys.json files
+// SigningKeyConfig returns the writable user level path of signingkeys.json files
 func (p *PathManager) SigningKeyConfig() string {
 	path, err := p.UserConfigFS.GetPath(SigningKeysFile)
 	checkError(err)
 	return path
 }
 
-// TrustPolicy returns the path of trustpolicy.json file
+// TrustPolicy returns the ready-only path of trustpolicy.json file
 func (p *PathManager) TrustPolicy() string {
 	path, err := p.ConfigFS.GetPath(TrustPolicyFile)
 	checkError(err)
 	return path
 }
```

Comment:
Instead of creating a separate function for reading and writing, can we write a function that takes UserLevel/SystemLevel as a param and determines the path internally?

Reply:
Updated.